packages/core/src/customer/strategies/apple-pay/apple-pay-customer-strategy.ts

```diff
@@ -323,14 +323,27 @@ export default class ApplePayCustomerStrategy implements CustomerStrategy {
               ]
             : [];
 
-        unselectedOptions?.forEach((option) =>
-            shippingOptions.push({
-                label: option.description,
-                amount: `${option.cost.toFixed(decimalPlaces)}`,
-                detail: option.additionalDescription,
-                identifier: option.id,
-            }),
-        );
+        unselectedOptions
+            ?.filter((option) => option.isRecommended)
+            ?.forEach((option) =>
+                shippingOptions.push({
+                    label: option.description,
+                    amount: `${option.cost.toFixed(decimalPlaces)}`,
+                    detail: option.additionalDescription,
+                    identifier: option.id,
+                }),
+            );
+
+        unselectedOptions
+            ?.filter((option) => !option.isRecommended)
+            ?.forEach((option) =>
+                shippingOptions.push({
+                    label: option.description,
+                    amount: `${option.cost.toFixed(decimalPlaces)}`,
+                    detail: option.additionalDescription,
+                    identifier: option.id,
+                }),
+            );
```

Comment:
Can we use `sortBy()` from `lodash` to avoid writing a similar block?

https://lodash.com/docs/4.17.15#sortBy
Here is a sample usage:
```
var users = [
  { 'user': 'fred',   'age': 48, flag:false },
  { 'user': 'barney', 'age': 36, flag:true },
  { 'user': 'fred',   'age': 40, flag:false },
  { 'user': 'barney', 'age': 34, flag:false }
];
 
_.sortBy(users, ['flag']).reverse();

// output: Array (4 items)
// 0: Object {user: "barney", age: 36, flag: true}
// 1: Object {user: "barney", age: 34, flag: false}
// 2: Object {user: "fred", age: 40, flag: false}
// 3: Object {user: "fred", age: 48, flag: false}
```

Reply:
@bc-peng I think there can be an ordering issue while sorting this way. Take following example:

```js

// Users from API
var users = [
  { 'user': 'fred',   'age': 30, flag: false },
  { 'user': 'barney', 'age': 32, flag: true },
  { 'user': 'fred',   'age': 36, flag: false },
  { 'user': 'barney', 'age': 40, flag: false }
];
 
_.sortBy(users, ['flag']).reverse();

// After sorting
// 0: Object {user: "barney", age: 32, flag: true}
// 1: Object {user: "barney", age: 40, flag: false}
// 2: Object {user: "fred", age: 36, flag: false}
// 3: Object {user: "fred", age: 30, flag: false}

// What we actually want
// 0: Object {user: "barney", age: 32, flag: true}
// 1: Object {user: "fred", age: 30, flag: false}
// 2: Object {user: "fred", age: 36, flag: false}
// 3: Object {user: "barney", age: 40, flag: false}
```

Options available in Control Panel for checkout settings:
![Screen Shot 2023-01-30 at 2 50 21 pm](https://user-images.githubusercontent.com/107909200/215383752-478ed93b-39a6-45be-8a79-60ef99cdb1e5.png)

I think, we would want to keep the ordering from the API response intact, apart from bumping the `isRecommended` options. 🤔

Reply:
In that case, maybe
```
shippingOptions = [
     unselectedOptions?.filter((option) => option.isRecommended),
     ...unselectedOptions?.filter((option) => option.!isRecommended),
]
```

Reply:
@bc-peng updated! 🙏